tasmota/language/ru_RU.h

```diff
@@ -84,7 +84,7 @@
 #define D_DATA "Данные"
 #define D_DARKLIGHT "Темный"
 #define D_DEBUG "Отладка"
-#define D_DEWPOINT "Dew point"
+#define D_DEWPOINT "Точка росы"
```

Comment:
Please make a separate PR for language support. Let's not mix topics.

Reply:
Okay, I will.